### PR TITLE
Create Analysis Tab

### PR DESCRIPTION
**Pull request by roshni-b**
New Analysis tab:
- identical to the Assay tab, but with updated headers and containing only the list assays that have support for analysis at present
- accessible to `cidc-biofx-user`  and `cidc-admin`

Goes with Docs [update](https://github.com/CIMAC-CIDC/cidc-documentation/pull/24) (separating assay & analysis mds)

### CODE REVIEW

File: src/components/info/InfoProvider.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import {
 export interface IInfoContext {
     supportedTemplates: {
         manifests: string[];
-        metadata: string[];
+        assays: string[];
```
Comment (jacoblurye) [inline src/components/info/InfoProvider.tsx:12]: This change is kind of tangential to the purpose of this PR - it makes the `InfoContext` more consistent with the `/info/assays` endpoint.

File: src/components/uploadDocs/UploadDocsPages.tsx [hunk 2]
```diff
@@ -0,0 +1,185 @@
+import * as React from "react";
```
Comment (jacoblurye) [inline src/components/uploadDocs/UploadDocsPages.tsx:1]: This file now contains a parametrized version of the old `AssaysPage` component that can display either Assay or Analysis docs based on the value of the `uploadType` prop.

File: src/components/uploadDocs/UploadInstructions.tsx [hunk 3]
```diff
@@ -30,9 +31,8 @@ const CopyIdToken: React.FunctionComponent = () => {
     );
 };
 
-const AssayInstructions: React.FunctionComponent<IAssayInstructionsProps> = props => {
-    const assay = props.match.params.assay;
-    const path = `cidc-documentation/master/assays/${assay}.md`;
```
Comment (jacoblurye) [inline src/components/uploadDocs/UploadInstructions.tsx:35]: We can no longer depend on the documentation living in the `assays/` subdirectory, so this component now takes a new prop `docPath` that should contain the path to the markdown document relative to the `cidc-documentation` repository root.

File: .env [hunk 4]
```diff
@@ -6,5 +6,5 @@
 # Staging config
 REACT_APP_AUTH0_DOMAIN=cidc-test.auth0.com
 REACT_APP_AUTH0_CLIENT_ID=Yjlt8LT5vXFJw1Z8m8eaB5aZO26uPyeD
-REACT_APP_API_URL=https://staging-api.cimac-network.org
+REACT_APP_API_URL=http://localhost:5000
```
Comment (jacoblurye) [inline .env:9]:
```suggestion
https://staging-api.cimac-network.org
```

File: .env [hunk 5]
```diff
@@ -6,5 +6,5 @@
 # Staging config
 REACT_APP_AUTH0_DOMAIN=cidc-test.auth0.com
 REACT_APP_AUTH0_CLIENT_ID=Yjlt8LT5vXFJw1Z8m8eaB5aZO26uPyeD
-REACT_APP_API_URL=https://staging-api.cimac-network.org
+https://staging-api.cimac-network.org
```
Comment (jacoblurye) [inline .env:9]:
```suggestion
REACT_APP_API_URL=https://staging-api.cimac-network.org 
```

File: .env [hunk 6]
```diff
@@ -6,5 +6,5 @@
 # Staging config
 REACT_APP_AUTH0_DOMAIN=cidc-test.auth0.com
 REACT_APP_AUTH0_CLIENT_ID=Yjlt8LT5vXFJw1Z8m8eaB5aZO26uPyeD
-REACT_APP_API_URL=https://staging-api.cimac-network.org
+REACT_APP_API_URL=https://staging-api.cimac-network.org
```
Comment (jacoblurye) [inline .env:9]:
```suggestion
REACT_APP_API_URL=https://staging-api.cimac-network.org
```

File: src/components/uploadDocs/UploadDocsPages.tsx [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+    );
+
+    const CLIRedirect = () => (
+        <Redirect to={`/${props.uploadType}/cli-instructions`}></Redirect>
+    );
+
+    const CLIUploadInstructions = () => (
+        <UploadInstructions
+            docPath={`/assays/cli-instructions.md`}
+            title={pathConfigs["cli-instructions"].title}
```
Comment (roshni-b) [inline src/components/uploadDocs/UploadDocsPages.tsx:94]:
```suggestion
            docPath={`/cli-instructions.md`}
```

File: src/components/uploadDocs/UploadDocsPages.tsx [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+        </ListItem>
+    );
+
+    const CLIRedirect = () => (
+        <Redirect to={`/${props.uploadType}/cli-instructions`}></Redirect>
+    );
+
+    const CLIUploadInstructions = () => (
+        <UploadInstructions
+            docPath={`/assays/cli-instructions.md`}
```
Comment (roshni-b) [inline src/components/uploadDocs/UploadDocsPages.tsx:93]:
```suggestion
            docPath={`/cli-instructions.md`}
```

**Pull request merged**